docs/react-native/features/tracking-error-component.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Tracking Render Errors
+description: Ensure meaningful screen names in render errors by configuring Metro bundler.
+sidebar_position: 7
+---
+
+# Tracking Render Errors
+
+To accurately track React Native rendering errors and include meaningful screen names in the `componentStack`, you need to configure your `metro.config.js` file. By default, in release builds, screen names may appear as `Unknown`. The following configuration ensures that class and function names are preserved during minification.
+
+## Implementation
+
+Add the following configuration to your `metro.config.js` file:
+
+```javascript
+const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
+
+const config = {
+  transformer: {
+    minifierConfig: {
+      keep_classnames: true,
```

Comment:
are both these needed? If the app only uses functional components is `keep_classnames` still required?